src/frametree.cpp

```diff
@@ -399,3 +399,21 @@ void FrameTree::cycle_frame(int delta) {
     focusFrame(frames[index]);
 }
 
+int FrameTree::cycle_frame_cmd(Input input, Output output) {
```

Comment:
Does this purposely deviate from our (new) naming scheme for members (`camelCase`)? If not, I'd suggest `cycleFrameCommand`. :-)

Reply:
I'm having trouble when I have to decide between naming the function like the command  and naming the function in camel case. Shall we agree on camel case as the naming convention? If yes, I suggest to fix the suffix `Cmd` for commands.

Reply:
I'm fine with `Cmd` as suffix, but keep in mind that all of the command functions that I have touched since we agreed on camelCase having been suffixed with `Command` (see RuleManager, TagManager, KeyManager, MouseManager). If you prefer `Cmd`, we should change it everywhere else, too. That should not stop us, though.

Reply:
Oh and: I've struggled with naming those methods, too. I ended up rather ignoring the command name when choosing a method name. So we have, for example, `addKeybindCommand` for `hc keybind` and `removeKeybindCommand` for `hc keyunbind`. Made more sense to me.

Reply:
I've renamed the commands in `FrameTree` to camelCase and added a `Command` suffix.